docs/qcreset.rst

```diff
@@ -0,0 +1,218 @@
+The qcreset module
+=====================
+
+The ``qcreset`` module provides methods (functions) to reset the selected data
+before starting a forward modelling workflow.
+
+
+Design philosophy
+-----------------
+
+* The client (user) scripts shall be small and simple and easy to use also
+  for modellers with little Python experience.
+* Input will be a python dictionary
+* The ``qcreset`` methods are meant to run inside RMS at the begining of
+  a modelling process
+
+
+set_data_empty
+--------------
+
+
+This method set all data described as input as empty. The data are not deleted
+so that the geometry of the data (surfaces or 3D grid properties) should be
+preserved and no workflow component (job or other) should be reset.
+
+
+Signature
+~~~~~~~~~
+
+The input of this method is a Python dictionary with defined keys. The key
+"project" is required while "horizons", "zones" and "grid_models" are optional
+(at least one of them should be provided for the method to have any effect).
+
+project
+  The roxar magic keyword ``project`` refering to the current RMS project.
```

Comment:
Since this is only inside RMS, "project" is not needed in the arguments list

Reply:
But if project is only called in the method (stored on github), you will get undefined variable pylint error isn't it ? Isn'it more elegant to parse it as argument instead of creating exception? What's your opinion on that ?

Reply:
Also, I guess the method needs to be available in RMS python in order to work, not komodo, isn't it ?

Reply:
@jcrivenaes :  no opinion on that topic ?

Reply:
Yes this must run inside RMS (or run in RMS licensed python shell). The issue with pylint complaining on "project" cannot be avoided. Of course you may think that at some stage the project is not the magic but a RMS project reference (if ran from terminal in a RMS python python shell), so... keep it for now and then we can evaluate when testing.